package io.github.cursodsousa.msavaliadorcredito.application.exception;

public class DadosClienteNotFoundException extends Exception{
    public DadosClienteNotFoundException() {
        super("Dados cliente n√£o encontrado para o cpf informado!");
    }
}
